core/trino-main/src/main/java/io/trino/sql/planner/planprinter/TextRenderer.java

```diff
@@ -135,6 +140,8 @@ private String printStats(PlanRepresentation plan, NodeRepresentation node)
         }
         output.append("\n");
 
+        printMetrics(output, "connector metrics:", nodeStats.connectorMetrics);
```

Comment:
use `getConnectorMetrics` instead (and below)

Reply:
Thanks, fixed.